### DISCUSSION TITLE
Does symfony adapt changes from PDO driver change in PHP 8.4

### DISCUSSION THREAD

**Discussion by HeenaBansal20** [symfony/symfony#58808] [Q&A]
HI , 
PHP 8.4 has [PDO driver changes ](https://wiki.php.net/rfc/pdo_driver_specific_subclasses)and introduced  [connect() method in PDO driver. ](https://github.com/php/php-src/commit/d6a0b3af68a55836ad7d24d4d832898c5b8c2d8e)
For my Symfony application , I stopped receiving spans for PDO __construct  call . I am just wondering if Symfony has updated its database connection call by replacing __construct with connect()?

I see the callbacks for PDO __construct in PHP 8.3 and connect in PHP 8.4.

symphony version: "v7.1.6"

**Comment by stof** [comment]
Symfony did not change anything, but Symfony only performs PDO connection in a few specific places (when using the PdoSessionHandler for instance).

However, I know that doctrine/dbal has introduced usage of the the `PDO::connect` method on PHP 8.4.
